orchestrator-salt/src/main/resources/salt/salt/postgresql/init.sls

```diff
@@ -24,14 +26,29 @@ init-services-db-remote:
 
 init-db-with-utf8:
   cmd.run:
-    - name: rm -rf /var/lib/pgsql/data && runuser -l postgres sh -c 'initdb --locale=en_US.UTF-8 /var/lib/pgsql/data > /var/lib/pgsql/initdb.log' && rm /var/log/pgsql_listen_address_configured
+    - name: rm -rf {{ postgres_data_directory }} && runuser -l postgres sh -c 'initdb --locale=en_US.UTF-8 {{ postgres_data_directory }} > /var/lib/pgsql/initdb.log' && rm /var/log/pgsql_listen_address_configured
     - unless: grep -q UTF-8 /var/lib/pgsql/initdb.log
 
+{%- if postgres_data_on_attached_disk %}
```

Comment:
Will it work for None? None is considered false?

Reply:
in theory this is always set if no remote db is configured. But would be safer to include `None`

Reply:
Using 'False; as default value in the beginning of the sls file